docs/Installation/BuildSystem.md

```diff
@@ -2,163 +2,57 @@
-excluded from coverage analysis by adding the inline comment `LCOV_EXCL_LINE`
-or a block can be excluded using `LCOV_EXCL_START` and `LCOV_EXCL_STOP`.
-However, this should be used extremely sparingly since unreachable code paths
-should be removed from the code base altogether.
 
-## Adding Executables {#adding_executables}
+where `FLAG{1..N}` are CMake flags that are detailed in the section on
+[commonly used CMake flags](#common_cmake_flags).
 
-All general executables are found in `src/Executables`, while those
-for specific evolution (elliptic) systems are found in
-`src/Evolution/Executables` (`src/Elliptic/Executables`).  See \ref
-dev_guide_creating_executables "how to create executables".
+CMake will look in your `$PATH` environment variable for all of the
+[build dependencies](installation.html#build_dependencies) and will use the
+first one it finds that satisfies the requirements. However, if a dependency is
+not in your `$PATH`, there are multiple versions of a dependency, or you just
+want to customize your configuration, you'll have to tell CMake which dependency
+you want to use and where it is using CMake flags. To that end, even though the
+[commonly used CMake flags](#common_cmake_flags) section has a more detailed
+list of the flags available, the following list has the *most common* CMake
+flags you'll need to customize your configuration (e.g. compilers, charm, build
+type, and library types):
 
-## Adding External Dependencies {#adding_external_dependencies}
+- CMAKE_C_COMPILER
+- CMAKE_CXX_COMPILER
+- CMAKE_Fortran_COMPILER
+- CHARM_ROOT
+- CMAKE_BUILD_TYPE
+- BUILD_SHARED_LIBS
 
-To add an external dependency, first add a `SetupDEPENDENCY.cmake`
-file to the `cmake` directory. You should model this after the
-existing one for `Brigand` if you're adding a header-only
-library and `yaml-cpp` if the library is not header-only. If CMake
-does not already support `find_package` for the library you're adding
-you can write your own. These should be modeled after `FindBrigand`
-for header-only libraries, and `FindYAMLCPP` for compiled
-libraries. The `SetupDEPENDENCY.cmake` file must then be included in
-the root `spectre/CMakeLists.txt`. Be sure to test both that setting
-`LIBRARY_ROOT` works correctly for your library, and also that if the
-library is required that CMake fails gracefully if the library is not
-found.
+\note If you are on a cluster, take a look at the
+[installation on clusters](installation_on_clusters.html)
+instructions. If you are on a cluster we support, we will already have an
+environment setup and an easy way for you to configure the build without having
+to specify flags yourself.
 
 ## Commonly Used CMake flags {#common_cmake_flags}
-The following are the most common flags used to control building with
-`CMake`. They are used by
-```
-cmake -D FLAG1=OPT1 ... -D FLAGN=OPTN <SPECTRE_ROOT>
-```
+The following are common flags used to control building SpECTRE with CMake (in
+alphabetical order):
```

Comment:
while you're touching the file, alphabetize the list (BUILD_DOCS, DOCS_ONLY, the CMAKE_*, SPECTRE_DEBUG)